rust-connectors/sinks/postgres/tests/simple.rs

```diff
@@ -0,0 +1,389 @@
+use fluvio::metadata::topic::TopicSpec;
+use fluvio_connectors_common::opt::CommonSourceOpt;
+use tokio::task::JoinHandle;
+use tokio::time::{sleep, Duration};
+use tokio_postgres::{Client, NoTls};
+use url::Url;
+
+#[tokio::test]
+async fn postgres_sink_and_source() -> eyre::Result<()> {
```

Comment:
I spent a bit of time trying to break this one massive serial test into 5 or more tests but it seems that they caused each other to break. The `start_pg_sink`, `start_pg_source` and `cleanup` functions are to enable okay setups and teardowns.